rest/taskana-rest-spring-example/src/main/java/pro/taskana/rest/JobScheduler.java

```diff
@@ -22,9 +30,13 @@
 @Component
 public class JobScheduler {
 
+    private final long untilDays = 14;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(JobScheduler.class);
     @Autowired
     private TaskanaEngine taskanaEngine;
+    @Autowired
+    private TaskService taskService;
 
     @Scheduled(fixedRate = 60000)
     @Transactional(rollbackFor = Exception.class)
```

Comment:
This @Transactional attribute must be removed. Otherwise, transaction slicing as introduced with TSK-511 would not work.